src/views/index.html

```diff
@@ -0,0 +1,87 @@
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"     import govukBackLink %}
+{% from "govuk/components/breadcrumbs/macro.njk"   import govukBreadcrumbs %}
+{% from "govuk/components/button/macro.njk"        import govukButton %}
+{% from "govuk/components/date-input/macro.njk"    import govukDateInput %}
+{% from "govuk/components/details/macro.njk"       import govukDetails %}
+{% from "govuk/components/error-message/macro.njk" import govukErrorMessage %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/footer/macro.njk"        import govukFooter %}
+{% from "govuk/components/header/macro.njk"        import govukHeader %}
+{% from "govuk/components/input/macro.njk"         import govukInput %}
+{% from "govuk/components/panel/macro.njk"         import govukPanel %}
+{% from "govuk/components/phase-banner/macro.njk"  import govukPhaseBanner %}
+{% from "govuk/components/radios/macro.njk"        import govukRadios %}
+{% from "govuk/components/summary-list/macro.njk"  import govukSummaryList %}
+{% from "govuk/components/table/macro.njk"         import govukTable %}
+{% from "govuk/components/textarea/macro.njk"      import govukTextarea %}
+{% from "govuk/components/file-upload/macro.njk"   import govukFileUpload %}
+{% from "govuk/components/inset-text/macro.njk"    import govukInsetText %}
+
+{% block head %}
+
+  <script src="https://code.jquery.com/jquery-1.12.4.min.js"
+          integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+          crossorigin="anonymous">
+  </script>
+
+  <div id="templateName" data-id='{{templateName}}' hidden></div>
+
+  {% block download %}
+  {% endblock %}
+{% endblock %}
+
+{% block header %}
+  {{ govukHeader({
+    homepageUrl: "/extensions",
+    serviceName: "Apply to extend your accounts filing deadline",
+    serviceUrl: "/extensions",
+    containerClasses: "govuk-width-container"})
+  }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "beta"
+    },
+    html: 'This is a new service – your <a class="govuk-link" href="https://www.research.net/r/extensions-feedback">feedback</a> will help us to improve it.'
```

Comment:
Please remove content or at least link href if it looks bad. There should be separate story and user need before we implement it.